packages-next/types/src/index.ts

```diff
@@ -215,6 +215,7 @@ export type BaseKeystone = {
   ) => BaseKeystoneList;
   connect: () => Promise<void>;
   lists: Record<string, BaseKeystoneList>;
+  createApolloServer: (a: { schemaName: string; dev: boolean }) => any;
```

Comment:
```suggestion
  createApolloServer: (args: { schemaName: string; dev: boolean }) => any;
```

afaik we've been using `args` everywhere we do this, lets be consistent